kolibri_instant_schools_plugin/auth/api.py

```diff
@@ -153,7 +154,7 @@ def create(self, request):
             resettoken.use_token()
 
             # change the password for all accounts associated with this phone number
-            set_password_for_phone(phone, password)
+            set_password_for_hashed_phone(hash_phone(phone), password)
```

Comment:
This seems to be a separate change? Looks like this was changed previously, but this reference missed?

Reply:
Ah yeah I ran into that as a bug while testing and forgot to mention it in the PR